extensions/wikia/Chat2/ChatController.class.php

```diff
@@ -112,6 +153,10 @@ public function onMakeGlobalVariablesScript( array &$vars ) {
 
 		$vars['wgChatLangMonthAbbreviations'] = $wgLang->getMonthAbbreviationsArray();
 
+		// SUS-2198
+		$vars['wgUseSiteJs'] = $this->wg->UseSiteJs == true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)